docs/page-caching.md

```diff
@@ -0,0 +1,79 @@
+# Page Caching
+
+The majority of the pages viewed on HM Cloud will be cached to improve performance and delivery time to the user. Its necessary to take into account the page cache when developing - considerations are listed at the bottom of this page. All pages are cached by default, and the following rules excluded specific cases:
+
+- All `POST` requests are not cached
+- Any requests with cookies matching the patterns `wordpress_*`, `wp-*`, `wp_*`, `comment_*` and `hm_*`are not cached.
+- Any requests with the `Authentication` HTTP header will not be cached.
+- Any response with the `Cache-Control: no-cache` HTTP header will not be cached.
+- Any response with no content are not cached.
+- Any request to `xmlrpc.php`, `wp-cron.php` or `wp-app.php` are not cached.
+- Any request to `wp-includes/js*` are not cached.
```

Comment:
Why js and not css too?

Reply:
I think this is a holdover, where some JS requests are routed to PHP, and say something to do with that.